fcrepo-kernel/src/main/java/org/fcrepo/kernel/observer/FedoraEvent.java

```diff
@@ -59,6 +65,7 @@
      */
     public FedoraEvent(final Event e) {
         checkArgument(e != null, "null cannot support a FedoraEvent!");
+        eventID = pidMinter.mintPid();
```

Comment:
Maybe we could inject a PidMinter instance instead of using the default constructor, allowing people to configure a minter specifically for events if they wanted to have a different id scheme than for nodes?